CHANGELOG.md

```diff
@@ -12,6 +12,14 @@ This upcoming release of the agent adds instrumentation for Active Support cachi
   | ----- | ----- | ----- |
   | `disable_active_support` | `false` | If `true`, disables Active Support instrumentation. |
 
+- **Add Action Dispatch Instrumentation**
```

Comment:
Maybe we should combine all the sections for the new instrumentation, since its all so similar, and the wording in the paragraphs is so similar as well.
We could just have a generic paragraph that describes all of it, and then just have a table underneath that lists all the different configs together.

Reply:
I like that a lot

Reply:
I like that too! let's do it.

Reply:
Created a new combined CHANGELOG entry with [df2a4c0](https://github.com/newrelic/newrelic-ruby-agent/pull/1745/commits/df2a4c00b3287fddffae8e5f407f06ca8fbfe6ac)